screens/CreateWorkout.js

```diff
@@ -165,20 +174,35 @@ class CreateWorkoutScreen extends React.Component {
 
   validateWorkout() {
     let exercises = this.state.exercises;
+    let sets = [];
     let exerciseNameMissing = false;
+    let setsDurationZero = false;
     let message = "";
     for (let i = 0; i < exercises.length; i++) {
+      for(let j=0; j<exercises[i].exerciseSets.length; j++) {
+        sets.push(exercises[i].exerciseSets[j]);
+      }
       if (exercises[i].name === "") {
         exerciseNameMissing = true;
       }
     }
 
+    for(let i = 0; i < sets.length; i++) {
+      if (sets[i].duration === 0) {
+        setsDurationZero = true;
+      }
+    }
+
+    console.warn(sets);
```

Comment:
You forgot this, it still shows the console message :)

Reply:
😬